src/ontology/cl-edit.owl

```diff
@@ -31484,6 +31487,36 @@ AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:27776108") oboInOwl:hasE
 AnnotationAssertion(rdfs:label obo:CL_4033040 "lung resident memory CD8-positive, CD103-positive, alpha-beta T cell")
 EquivalentClasses(obo:CL_4033040 ObjectIntersectionOf(obo:CL_4033039 ObjectSomeValuesFrom(obo:RO_0002104 obo:PR_000001010)))
 
+# Class: obo:CL_4033041 (alveolar macrophage CCL3-positive)
+
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:37291214") obo:IAO_0000115 obo:CL_4033041 "An alveolar macrophage that expresses CCL3.")
```

Comment:
Consider adding logical statement to represent "expresses CCL3".
If there are any other distinguishing features (e.g., marker genes) or functions ((e.g., see doi:[10.1128/IAI.69.10.6364-6369.2001](https://doi.org/10.1128/iai.69.10.6364-6369.2001)) consider adding.

Reply:
Thank you for your comment. The ontology that is normally used is PR, but in this case the [CCL3 (human)](https://www.ebi.ac.uk/ols4/ontologies/pr/classes/http%253A%252F%252Fwww.genenames.org%252Fcgi-bin%252Fgene_symbol_report%253Fhgnc_id%253D10627) term is imported, and I am not sure if I should use it.
I have added the markers that the reference provided in Supplementary Table 6 (see screenshot):
![image](https://github.com/obophenotype/cell-ontology/assets/94959119/af3c1c73-d9c3-48b7-a941-d1955cc9ff0f)

Interestingly, these markers differ from other sources (eg. Azimuth):
<img width="1033" alt="image" src="https://github.com/obophenotype/cell-ontology/assets/94959119/79d3ead0-7e61-4dc8-9a2e-d280d4a46d73">

> If there are any other distinguishing features (e.g., marker genes) or functions ((e.g., see doi:[10.1128/IAI.69.10.6364-6369.2001](https://doi.org/10.1128/iai.69.10.6364-6369.2001)) consider adding to definition or comment.

I am reluctant to use conclusions of this study in this term. In the study, authors use KO mice, meaning that they are comparing alveolar macrophages WT mice, with alveolar macrophages CCL3-/-. The conclusion is that KO mice have impaired phagocytic activity respect WT. In my opinion, it doesn't mean that this cell has more phagocytic activity than other WT alveolar macrophages, as that could only be demonstrated using cell sorting from a WT mouse and test the phagocytic activity. As I might have missed something from the article, please let me know if you agree with my interpretation.